nodejs/v3/nestjs-prisma/app/package.json

```diff
@@ -21,7 +21,7 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@appsignal/nodejs": "^3.0.12",
+    "@appsignal/nodejs": "file:../integration",
```

Comment:
If you set the path like this, the `npm link` command is not necessary. It probably works because of this and not the npm link change. I would expect `npm install` to overwrite the link again.

Reply:
Hm 🤔 But I didn't set that manually -- running `npm link --save --package-lock` did that.

What do you mean that `npm install` would override the link? `npm install` runs [right after the link is established](https://github.com/appsignal/test-setups/pull/130/files#diff-3a5dca3361f5057546ab5986e1dce86ac5c73733a60142f2c3466ab309414f33R20-R24).

Reply:
> Hm 🤔 But I didn't set that manually -- running `npm link --save --package-lock` did that.

I think the `--save` flag does that.

> What do you mean that `npm install` would override the link? `npm install` runs [right after the link is established](https://github.com/appsignal/test-setups/pull/130/files#diff-3a5dca3361f5057546ab5986e1dce86ac5c73733a60142f2c3466ab309414f33R20-R24).

In your latest changes, yes. In all our other scripts (and this one before your change) `npm install` overwrites `npm link` when I've tried to study this behavior.

Reply:
Should I undo this change then @tombruijn? I'm unsure how to proceed. I suppose that now that it's picked up the new dependencies and updated the `package-lock.json` file, it would be fine to not set the path explicitly in the `package.json`.

Reply:
I'll give that a try and commit that if it works. 👍

Reply:
Running it without `--save --package-lock` change in `npm link` did not undo the changes to the `package-lock.json`.

Reply:
Managed to sort it out, mostly by editing the files manually and running the test setup to see if it caused any changes on them: https://github.com/appsignal/test-setups/pull/130/commits/22102892b36c0f75af89d4311741d4d9e5e65130